packages/gatsby-plugin-manifest/src/gatsby-node.js

```diff
@@ -9,7 +9,7 @@ function generateIcons(icons, srcIcon) {
   return Promise.map(icons, icon => {
     const size = parseInt(icon.sizes.substring(0, icon.sizes.lastIndexOf(`x`)))
     const imgPath = `./public/` + icon.src
```

Comment:
Could you have this use [path.join](https://nodejs.org/api/path.html#path_path_join_paths) as well?

Reply:
@KyleAMathews fixed it! 😇